.github/workflows/pr-build.yaml

```diff
@@ -29,15 +29,89 @@ jobs:
     runs-on: ubuntu-latest
     permissions:
       contents: read
+    env:
+      CAPACT_MANIFEST_REPOSITORY: git::${{ github.event.pull_request.head.repo.clone_url }}
+      CAPACT_MANIFEST_BRANCH: ${{ github.event.pull_request.head.sha }}
+      CAPACT_MANIFEST_WAITING_ITERATIONS: "100"
+      CAPACT_CREDENTIALS_STORE_FILE_PASSPHRASE: password
+      CAPACT_CREDENTIALS_STORE_BACKEND: file
+      CAPACT_GATEWAY_HOST: https://gateway.capact.local
+      CAPACT_GATEWAY_USERNAME: graphql
+      CAPACT_GATEWAY_PASSWORD: okon
+      CAPACT_INSTALL_COMPONENTS: neo4j,ingress-nginx,cert-manager,capact
```

Comment:
This is minimal setup needed to have Capact Gateway and Hub working properly. We won't run Actions, so I don't install Argo, kubed, etc.